zerver/data_import/mattermost.py

```diff
@@ -389,13 +425,21 @@ def message_to_dict(post_dict: Dict[str, Any]) -> Dict[str, Any]:
         else:
             reactions = []
 
-        return dict(
+        message_dict = dict(
             sender_id=sender_id,
-            receiver_id=post_dict["channel"],
             content=content,
             pub_date=int(post_dict['create_at'] / 1000),
             reactions=reactions
         )
+        if "channel" in post_dict:
+            message_dict["channel_name"] = post_dict["channel"]
+        elif "channel_members" in post_dict:
```

Comment:
Let's add a comment explaining that this case is for private message conversations, not actually a channel.

Reply:
Done.